packages/internal/src/build/babel/web.ts

```diff
@@ -76,7 +76,12 @@ export const getWebSideBabelPlugins = (
       'rwjs-web-auto-import',
     ],
     ['babel-plugin-graphql-tag', undefined, 'rwjs-babel-graphql-tag'],
-    // === Handling redwood "magic"
+    process.env.NODE_ENV !== 'development' && [
```

Comment:
I would have kept the logic inside the plugin - as in always run the plugin, but the plugin doesn't do anything if its not in dev.